.nvmrc

```diff
@@ -1 +1 @@
-13.13.0
+15
```

Comment:
I'd recommend pinning this to a minor version. Additionally, it may make sense to use v16, as it's the current version: https://nodejs.org/en/about/releases

Reply:
Done